en/integrations/plugin-packs/procedures/applications-drbd-ssh.md

```diff
@@ -0,0 +1,172 @@
+---
+id: applications-drbd-ssh
+title: DRBD SSH
+---
+
+## Overview
+
+DRBD (Distributed Replicated Block Device) is a distributed storage architecture for GNU/Linux, 
+allowing the replication of block devices (disks, partitions, logical volumes etc.) between servers. DRBD is free software, but support exists. DRBD consists of a kernel module, 
+administration tools in user space as well as shell scripts.
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+* Resources
+
+### Collected Metrics
+
+More information about collected metrics is available in the official DRBD SSH documentation : https://www.linbit.com/drbd-user-guide/drbd-guide-9_0-en/
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Resources-->
+
+| Metric name                         | Description                         | Unit  |
+| :-----------------------------------| :-----------------------------------| :---- |
+| disk-status                         | Disk status                         |       |
+| peer-connection-status              | Peer connection status              |       |
+| peer-device-replication-status      | Peer device replication status      |       |
+| peer-device-disk-status             | Peer device disk status             |       |
+| resources.total.count               | Total number of resources           | count |
+| disk.data.read.bytespersecond       | Disk data read                      |  B/s  |
+| disk.data.written.bytespersecond    | Disk data written                   |  B/s  |
+| peer.traffic.in.bitspersecond       | Peer traffic in                     |  b/s  |
+| peer.traffic.out.bitspersecond      | Peer traffic out                    |  b/s  |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+A number of distributions provide DRBD SSH, including pre-built binary packages. 
+Support for these builds, if any, is being provided by the associated distribution vendor. 
+Their release cycle may lag behind DRBD SSH source releases.
+
+More information is available on the official documentation of DRBD SSH : https://www.linbit.com/drbd-user-guide/drbd-guide-9_0-en/#ch-install-packages
```

Comment:
@psamecentreon you must describe prerequisites for the Plugin to work. I do not understand this part, do we need this specific DRDB SSH extension? 

If so, you can probably also explained that a SSH connection have to be allowed between poller and DRBD system. (Look at BlueMind monitoring proc as an example)